src/test/java/com/scottlogic/filters/AuthorFilterTest.java

```diff
@@ -91,14 +91,12 @@ public void filter_nameNotInList_emptyReturned() {
         List<UserPost> actualResult = filterByAuthor.filter(userPosts);
```

Comment:
You can see that we probably didn't do the best job in the merge here as the spacing has gone a bit out of whack. But it's not actually done any harm really, so it's probably not worth fixing